src/locales/en/fields.yml

```diff
@@ -141,6 +162,9 @@ selfAssessmentPaymentAmount:
   label: Payment amount
   hint: Enter the amount in pounds and pence. For example, 3625.75.
   content: ""
+  validation:
+    required: Enter an amount
+    regex: Enter a valid amount in pounds and pence, like 123.45
```

Comment:
According to the AC there should be a separate message for inputs with missing demical - “Amount must include pounds and pence, like 123.45”

So I think we need to match for both:
- invalid characters/format ie. not numbers -> "Enter a valid amount in pounds and pence, like 123.45"
- a valid amount eg. "123" but missing decimal -> "Amount must include pounds and pence, like 123.45"